build.gradle

```diff
@@ -740,9 +740,21 @@ tasks.register('generateCSharpCodecsTests', JavaExec) {
             'sbe-benchmarks/src/main/resources/fix-message-samples.xml']
 }
 
+tasks.register('generateCSharpTestDtos', JavaExec) {
+    mainClass.set('uk.co.real_logic.sbe.SbeTool')
+    classpath = project(':sbe-tool').sourceSets.main.runtimeClasspath
+    systemProperties(
+            'sbe.output.dir': 'csharp/sbe-generated',
+            'sbe.target.language': 'uk.co.real_logic.sbe.generation.csharp.CSharpDtos',
```

Comment:
I think this is better than the approach for multiple generators in https://github.com/real-logic/simple-binary-encoding/pull/951. 

For that one, we added an extra flag `sbe.go.generate.generate.flyweights=true`. Thoughts?

Reply:
I struggled to choose between the two mechanisms: flag vs. generator. I opted for the generator, as it seemed less intrusive (and less likely to collide with changes in my other PR). However, at that time, I had not noticed the existing pattern for the Go generation. Using a flag avoids spinning up a new JVM and parsing the schema multiple times. Therefore, that might be a better pattern. We should be consistent in any case. I'm happy to use a flag instead.

Reply:
For consistency, I've added a system property, `sbe.csharp.generate.dtos`, that enables DTO generation when using the `CSharp` rather than `CSharpDtos` target.